src/components/App/App.jsx

```diff
@@ -56,6 +60,30 @@ const Operations = Object.freeze({
   Erase: Symbol("erase")
 })
 
+// get best recipe matches with user's food items
+useEffect(() => {
+  async function fetchData() {
+    try {
+      // API parameter format: ingredient,+ingredient,+ingredient
+      let ingredientParams = ""
+      for (let i = 0; i < products.length; i++) {
+        if (i == 0) {
+          ingredientParams += products[i].name
+        } else {
+          ingredientParams += `,+${products[i].name}`
+        }
+      }
```

Comment:
You should be able to accomplish this with something like this:
```
products.join()
```
That should take an array of items and join it with a separator. The default separator is `,`

Reply:
Thanks for the suggestion, this works! I changed it to
`let ingredientParams = products.map((product) => (product.name)).join(",+")`